resources/user.py

```diff
@@ -4,25 +4,41 @@
 from passlib.apps import custom_app_context as pass_context
 from models import dtlogin, dtusers
 
+from flask_jwt_extended import (create_access_token,create_refresh_token,
+jwt_required, jwt_refresh_token_required,get_jwt_identity, get_raw_jwt)
 
 auth = HTTPBasicAuth()
 
+#parsing incoming data
+parser = reqparse.RequestParser()
+parser.add_argument('username', help = 'Please fill in the username', required = True)
+parser.add_argument('password', help= 'This field cannot be blank', required = True)
 class UserResource(Resource):
     """get all registered users"""
     def get(self):
         return dtusers, 200
 
     """create new user"""
     def post(self):
-        json_data = request.get_json(force=True)
-        
-        if not json_data:
-            return {'message': 'No input data provided'}, 400
-        elif (len(json_data)!=len(dtusers[0])):
-            return "Please enter all details"
+        #json_data = request.get_json(force=True)
+        data = parser.parse_args()        
+    
         
-        dtusers.append(json_data)
-        return {"status":"success", "data": json_data }, 201
+        try:
+            dtusers.append(data)
```

Comment:
since you have already created your db and tables you could just save the data directly to the db

Reply:
Before you create a new user, you need to query the user table to make sure that the user doesn't already exist in the db, also do more data validation to ensure that a user doesn't input empty strings etc